.baseline/checkstyle/checkstyle.xml

```diff
@@ -19,6 +19,7 @@
         <property name="lineSeparator" value="lf"/>
     </module>
     <module name="RegexpMultiline"> <!-- Development Practices: Writing good unit tests -->
+        <property name="id" value="VisibleForTestingPackagePrivate"/>
```

Comment:
you actually need to make this change to the other checkstyle.xml

Reply:
🤦‍♂

Reply:
Done